pkg/fixtures/fixtures.go

```diff
@@ -269,18 +269,52 @@ func (fxt *Fixture) parseArray(params []interface{}, parent string, index int) [
 
 func (fxt *Fixture) parseQuery(value string) string {
 	// Queries to fill data will start with #$ and contain a : -- search for both
-	// to make sure that we're trying to parse a query
-	r := regexp.MustCompile(`\${(.+):(.+)}`)
+	// to make sure that we're trying to parse a query.
+	// Additionally look for an optional default value: ${name:json_path|default_value}
+	r := regexp.MustCompile(`\${([^\|]+):([^\|]+)\|?(.+)?}`)
 	if r.Match([]byte(value)) {
 		nameAndQuery := r.FindStringSubmatch(value)
 		name := nameAndQuery[1]
 
+		// Check if there is a default value specified
+		if nameAndQuery[3] != "" {
```

Comment:
Do you need to check the length here? (want to make sure we don't get any array out of bounds errors)

Reply:
@tomer-stripe AFAIK the regex always returns three groups (https://play.golang.org/p/pf_oViIZBah), so we should be good unless we're concerned that this behaviour might change in Go?